src/app.js

```diff
@@ -78,6 +78,7 @@ app.use(
 
 app.use(setLocals);
 app.use(cspHandler);
+app.use(loggerMiddleware);
```

Comment:
I think we include the session ID in the logs - do we need this to come after the session configuration?